<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RCS2000</class>
 <widget class="QWidget" name="RCS2000">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>90</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="toolTipDuration">
   <number>0</number>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QGroupBox" name="Rotation">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>0</y>
     <width>411</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="title">
    <string>Rotation</string>
   </property>
   <widget class="QGroupBox" name="Pitch">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="title">
     <string>Pitch</string>
    </property>
    <widget class="QPushButton" name="PitchUp">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="PitchDown">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="PitchThrust">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Yaw">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="title">
     <string>Yaw</string>
    </property>
    <widget class="QPushButton" name="YawLeft">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="YawRight">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="YawThrust">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Roll">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="title">
     <string>Roll</string>
    </property>
    <widget class="QPushButton" name="CW">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="CCW">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="RollThrust">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="Translation">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>0</y>
     <width>401</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="title">
    <string>Translation</string>
   </property>
   <widget class="QGroupBox" name="X">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="title">
     <string>Up/Down</string>
    </property>
    <widget class="QPushButton" name="TransUp">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(6, 6, 6);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="TransDown">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="UpDwnThrust">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Y">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="title">
     <string>Port/Starboard</string>
    </property>
    <widget class="QPushButton" name="TransPort">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="TransStarboard">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="PrtStbThrust">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Z">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="title">
     <string>Forward/Reverse</string>
    </property>
    <widget class="QPushButton" name="TransForward">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="shortcut">
      <string>F</string>
     </property>
    </widget>
    <widget class="QPushButton" name="TransReverse">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="shortcut">
      <string>R</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="FwdRevThrust">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>20</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>RCS 2000</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>60</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 255, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>PitchUp</sender>
   <signal>clicked()</signal>
   <receiver>RCS2000</receiver>
   <slot>PitchUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PitchDown</sender>
   <signal>clicked()</signal>
   <receiver>RCS2000</receiver>
   <slot>PitchDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>YawLeft</sender>
   <signal>clicked()</signal>
   <receiver>RCS2000</receiver>
   <slot>YawLeft()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>YawRight</sender>
   <signal>clicked()</signal>
   <receiver>RCS2000</receiver>
   <slot>YawRight()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CW</sender>
   <signal>clicked()</signal>
   <receiver>RCS2000</receiver>
   <slot>RollClockWise()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CCW</sender>
   <signal>clicked()</signal>
   <receiver>RCS2000</receiver>
   <slot>RollCounterClockWise()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TransUp</sender>
   <signal>clicked()</signal>
   <receiver>RCS2000</receiver>
   <slot>TranslateUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>600</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TransDown</sender>
   <signal>clicked()</signal>
   <receiver>RCS2000</receiver>
   <slot>TranslateDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>630</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TransPort</sender>
   <signal>clicked()</signal>
   <receiver>RCS2000</receiver>
   <slot>TranslatePort()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>730</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TransStarboard</sender>
   <signal>clicked()</signal>
   <receiver>RCS2000</receiver>
   <slot>TranslateStarboard()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>760</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TransForward</sender>
   <signal>clicked()</signal>
   <receiver>RCS2000</receiver>
   <slot>TranslateForward()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>860</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TransReverse</sender>
   <signal>clicked()</signal>
   <receiver>RCS2000</receiver>
   <slot>TranslateReverse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>890</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Exit()</slot>
  <slot>PitchUp()</slot>
  <slot>PitchDown()</slot>
  <slot>YawRight()</slot>
  <slot>YawLeft()</slot>
  <slot>RollClockWise()</slot>
  <slot>RollCounterClockWise()</slot>
  <slot>TranslateUp()</slot>
  <slot>TranslateDown()</slot>
  <slot>TranslatePort()</slot>
  <slot>TranslateStarboard()</slot>
  <slot>TranslateForward()</slot>
  <slot>TranslateReverse()</slot>
 </slots>
</ui>
